src/App.tsx

```diff
@@ -1,7 +1,33 @@
 import React from 'react';
 
-export const App: React.FC = () => (
-  <div className="App">
-    <p className="App__message">The last pressed key is [Enter]</p>
-  </div>
-);
+type State = {
+  pressedKey: string,
+};
+
+export class App extends React.Component<{}, State> {
+  state: State = {
+    pressedKey: 'Nothing was pressed yet',
+  };
+
+  componentDidMount(): void {
+    document.addEventListener('keyup', this.keyClick);
+  }
+
+  componentWillUnmount(): void {
+    document.removeEventListener('keyup', this.keyClick);
+  }
+
+  keyClick = (event: KeyboardEvent): void => {
```

Comment:
1 rule in checklist
https://github.com/mate-academy/react_keyboard/blob/master/checklist.md